rpc/core/blocks.go

```diff
@@ -312,3 +312,66 @@ func Commit(heightPtr *int64) (*ctypes.ResultCommit, error) {
 	commit := blockStore.LoadBlockCommit(height)
 	return ctypes.NewResultCommit(header, commit, true), nil
 }
+
+// BlockResults gets ABCIResults at a given height.
+// If no height is provided, it will fetch the latest block.
+//
+// Results are for the tx of the last block with the same index.
+// Thus response.results[5] is the results of executing
+// getBlock(h-1).Txs[5]
+//
+// ```shell
+// curl 'localhost:46657/block_results?height=10'
```

Comment:
re ^^: so here we're fetching results for height=9, right?

Reply:
changed this. now you get what you asked for